examples/creatives/banner_nonlinear.js

```diff
@@ -0,0 +1,61 @@
+class BannerNonLinear extends BaseSimidCreative{
+    constructor() {
+        super();
+        this.addButtonClickActions_();
+    }
+
+    /**
+     * Receives init message from the player.
+     * @param {!Object} eventData Data from the event.
+     * @protected
+     */
+    onInit(eventData) {
```

Comment:
add override here. i think if you add that, you won't need the rest of the jsdoc from the base simid creative here

Reply:
Added